packages/vuetify/src/components/VSpeedDial/VSpeedDial.js

```diff
@@ -68,6 +73,12 @@ export default {
 
         return b
       })
+
+      if (!this.hideOverlay) {
```

Comment:
It should not display the overlay when opened on hover

Reply:
working on it